.github/workflows/jira.yml

```diff
@@ -2,6 +2,8 @@
 name: Jira Sync
 
 on:
+    workflow_dispatch:
+
```

Comment:
We should have at least one trigger here.
this should be deleted once the jira integration is ready